packages/contracts-bedrock/src/universal/StandardBridgeExtension.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { StandardBridge } from "./StandardBridge.sol";
+
+abstract contract StandardBridgeExtension is StandardBridge {
```

Comment:
Can you add some natspec like this to explain why this contract exists? 

https://github.com/ethereum-optimism/optimism/blob/c7c8930b9b37e5937057f8c34a9385c74a263aec/packages/contracts-bedrock/src/universal/StandardBridge.sol#L15